test/unified_format.py

```diff
@@ -883,6 +887,17 @@ def _databaseOperation_listCollections(self, target, *args, **kwargs):
         cursor = target.list_collections(*args, **kwargs)
         return list(cursor)
 
+    def _databaseOperation_createCollection(self, target, *args, **kwargs):
+        # PYTHON-1936 Ignore the listCollections event from create_collection.
```

Comment:
I had to add this workaround to fix the listCollections/createCollection problem described here: https://github.com/mongodb/specifications/pull/1202#discussion_r863158569